Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGrid.cs

```diff
@@ -5768,7 +5769,17 @@ private void DataGrid_LostFocus(object sender, RoutedEventArgs e)
                     focusedDependencyObject = parent;
                 }
 
-                if (focusLeftDataGrid)
+                if (this.EditingRow != null && this.EditingColumnIndex != -1)
+                {
+                    editingColumn = this.ColumnsItemsInternal[this.EditingColumnIndex];
+
+                    if (focusLeftDataGrid && editingColumn is DataGridTemplateColumn)
+                    {
+                        dataGridWillReceiveRoutedEvent = false;
```

Comment:
@XAML-Knight @michael-hawker Wouldn't it make more sense to simply return here, rather than hook into the LostFocus event below? I don't think that behavior works the way I thought it would.

Reply:
@duke7553, feel free to submit an issue for this.  FWIW, the unit tests pass if we simply returned from here, and the behavior of the DataGrid appears to remain the same, as we'd expect.